docs/development/running-the-dev-environment.md

```diff
@@ -1,154 +1,91 @@
-master branch. If you are not familiar with git clone, this will download a copy of
-Baserow's code to your computer.
+Backend and frontend run natively. Docker only for PostgreSQL/Redis.
 
-```
-$ cd ~/baserow
-$ git clone --branch master https://github.com/baserow/baserow.git
-Cloning into 'baserow'...
-...
-$ cd baserow
+```bash
+git clone --branch develop https://github.com/baserow/baserow.git
+cd baserow
+just init       # First time: install dependencies
+just dev up     # Start all services (Ctrl+C to stop)
 ```
 
-Now that we have our copy of the repo and have changed directories to the newly
-created `baserow`, we can bring up the containers. You just have to execute the
-docker-compose command using the `docker-compose.yml` file. It might take a while for
-the command to finish, this is because the images have to be built from scratch.
+Or run in background:
 
+```bash
+just dev up -d  # Start in background
+just dev logs   # View logs
+just dev stop   # Stop services
 ```
-$ ./dev.sh 
-Building backend
-...
-Starting db    ... done
-Starting redis    ... done
-Starting backend    ... done
-Starting web-frontend   ... done
-```
-
-Your dev environment is now running, the database has been automatically migrated for
-you and the baserow templates have been synced. You can now visit http://localhost:3000
-to sign up and login to your Baserow.
 
-## Looking at the web api
+Access http://localhost:3000 to start using Baserow.
 
-Baserow's backend container exposes a rest API. Find the API spec for your local version
-of Baserow at http://localhost:8000/api/redoc/ . To check that it is working correctly
-when you visit http://localhost:8000/api/workspaces/ in a browser you should see the error
-"Authentication credentials were not provided." as no JWT was provided.
+See [Running Locally](running-the-dev-env-locally.md) for details.
 
-## Attaching to the dev environment
+## Prerequisites
 
-The dev environment consists of a number of docker containers, see:
+### Minimal (Docker only)
 
-If you use `./dev.sh` by default it will attempt to open tabs in your terminal and
-attach to the running baserow containers. Otherwise you can do so manually by running
-the following commands:
+- Docker Desktop (or Docker Engine + Compose)
+- Git
+- just
 
-```bash
-$ # Run the commands below to connect to the various different parts of Baserow
-$ docker attach backend
-$ docker attach celery 
-$ docker attach web-frontend
-```
+### Full (Local development)
 
-When attached you can press CTRL-C to end the current containers main process. However
-unlike normal docker containers this one will not exit immediately but instead present
-you with a bash terminal. In this terminal you can then run any admin commands you wish
-or inspect the state of the containers. Simply press up and go through the containers
-bash history to get the original command to restart the containers main process.
+- All of the above, plus:
+- Python 3.11 + uv
```

Comment:
Again, I think the less places we mention specific versions and requirements, the better.